mhkit/river/io/d3d.py

```diff
@@ -6,6 +6,86 @@
 import numpy as np
 import pandas as pd
 import netCDF4
+import warnings
+
+
+def get_all_timestamps (data):
+    '''
+    retuns all of the time stamps and their associated time index
+    Parameters
+    ----------
+    data: netcdf4 object 
+       A netCDF object that contains spatial data, e.g. velocity or shear stess.
+       These can be generated by running a Delft3D model.  
+
+    Returns
+    -------
+      time_stamps: array
+        An array of positive interger or float the repesents the amount of time in seconds 
+        that simulation has been running
+
+    '''
+    assert type(data)== netCDF4._netCDF4.Dataset, 'data must be netCDF4 object'
+        
+    max_time= data['time'].shape
+    max_time= int(''.join(map(str, max_time)))
+    time_indexes= np.linspace(0, max_time-1, num=max_time)
+        
+    time_stamps= []        
+    for i in time_indexes:
+        time= np.ma.getdata(data.variables['time'][i], False)
+        time_stamps= np.append(time_stamps, time)
```

Comment:
Isn't this entire function just doing this:
`np.ma.getdata(data.variables['time'][:], False)`

Reply:
Yes it is. Do you think it's worth having a whole function for the one line of code?